app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+from time import sleep
+from datetime import datetime
+
+
+def loging():
+    while True:
+        with open(datetime.now().strftime("app-%H_%M_%S.log"), "w+") as file:
+            file.write(str(datetime.now()))
```

Comment:
You forgot to add this - `The app must print to console timestamp and newly created file name when it completes file creation successfully.`

Reply:
Thx)